hack/generate-kustomize-patches.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
Script for generating watchfilter kustomize patches, because these can change without us noticing it.